src/main/java/core/basesyntax/service/impl/WritingFileServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WritingFileService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class WritingFileServiceImpl implements WritingFileService {
+    @Override
+    public void writingDataToFile(String reportData, String outputFile) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(outputFile, true))) {
+            Files.writeString(Path.of(outputFile), "");
```

Comment:
Use **only** nio if you use it here and in ReadingFileServiceImpl

Reply:
I don't quite understand how to do it. I used the logic similar to the one described in the task, without changing anything:
https://github.com/mate-academy/jv-work-with-file/pull/1013/files